packages/salesforce-adapter/src/change_validators/multiple_deafults.ts

```diff
@@ -0,0 +1,110 @@
+/*
+*                      Copyright 2021 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import {
+  ChangeError, getChangeElement, ChangeValidator, isAdditionOrModificationChange,
+  isInstanceChange, Field, InstanceElement,
+  Element, isFieldChange, isListType, TypeElement, isMapType, Value, isReferenceExpression,
+} from '@salto-io/adapter-api'
+import { values } from '@salto-io/lowerdash'
+import { safeJsonStringify } from '@salto-io/adapter-utils'
+import _ from 'lodash'
+import { LABEL } from '../constants'
+import { apiName, isFieldOfCustomObject } from '../transformers/transformer'
+
+const fieldNameToInnerContextField: Record<string, string> = {
+  applicationVisibilities: 'application',
+  recordTypeVisibilities: 'recordType',
+  standardValue: 'label',
+  customValue: 'label',
+}
+
+const formatContext = (context: Value): string => {
+  if (isReferenceExpression(context)) {
+    return context.elemId.getFullName()
+  } if (_.isString(context)) {
+    return context
+  } return safeJsonStringify(context)
```

Comment:
nit: missing line breaks
```suggestion
  if (isReferenceExpression(context)) {
    return context.elemId.getFullName()
  }
  if (_.isString(context)) {
    return context
  }
  return safeJsonStringify(context)
```